docs/user-guide/introspection-and-debugging.md

```diff
@@ -181,6 +181,15 @@ Here you can see the event generated by the scheduler saying that the Pod failed
 
 To correct this situation, you can use `kubectl scale` to update your Replication Controller to specify four or fewer replicas. (Or you could just leave the one Pod pending, which is harmless.)
 
+Events such as ones you saw at the end of `kubectl describe pod` are persisted in etcd and provide high-level information on what is happening in the cluster. To list all events you can use
+```
+kubectl get events
+```
+but you have to remember that events are namespaced. This means that if you're interested in events for some namespaced object (e.g. what happened with Pods in namespace `my-namespace`) you need to explicitly provide a namespace to the command:
+```
+kubectl get events --namespace=my-namespace
+```
```

Comment:
Can you add

To see events from all namespaces, you can use the `--all-namespaces` argument